src/bundler.ts

```diff
@@ -267,12 +282,13 @@ class Bundler {
     // should be removed from the document.
     const importMap: Map<string, Import|null> = new Map();
     analyzedRoot.getByKind('import').forEach((i) => importMap.set(i.url, i));
-    this.excludes.forEach((u) => {
-      if (importMap.has(u)) {
+    importMap.forEach((_, u) => {
```

Comment:
This is n^2, but I'm not sure how to avoid it if we consider excludes by suffix.